app/templates/collections/collection/edit.hbs

```diff
@@ -1,59 +1,31 @@
-<div class="coll-add-layer p-xl">
-    <div class="container">
-        {{# if editMode}}
+{{# if editMode}}
```

Comment:
I dono what happened to this file. Need to chat about the merge conflicts here. edit looks really poor rn.